### PR TITLE
Added landing message

### PR DESCRIPTION
**Pull request by rahulgpote**
Task done of an issue

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
   <title>Welcome to Ecom/title>
   </head>
   <body>
+	<h1> Welcome to Ecom! Enjoy your stay </h1>
```
Comment (rahulgpote) [inline index.html]: Could you remove the 'Enjoy your stay. string and make the 'Welcome to Ecom!" string red?

Comment (rahuldevlp) [reply index.html]: ok i will update

**Review by rahulgpote — COMMENTED** [review]
nice work

**Review by rahulgpote — COMMENTED** [review]
done

**Pull request merged**